plugins/Live/Model.php

```diff
@@ -509,4 +531,25 @@ private function getWhereClauseAndBind($whereClause, $bindIdSites, $idSite, $per
         }
         return array($whereBind, $where);
     }
+
+    /**
+     * @param $ecommerceIdGoal
+     * @return string
+     */
+    private function getSqlEcommerceConversionsLifeTimeMetricsForIdGoal($ecommerceIdGoal)
+    {
+        $sql = "SELECT
+                    COALESCE(SUM(" . LogAggregator::getSqlRevenue('revenue') . "), 0) as lifeTimeRevenue,
```

Comment:
`SUM` returns `NULL` when there are not matching rows (ie. no ecommerce conversions)